### ISSUE TITLE
Windows 10 "access is denied" error for spiral drawing program.

### ISSUE THREAD

**Issue by asweigart** [asweigart/pyautogui#52]
This spiral drawing program (which uses mouse drags to draw in MSPaint) results in a "error 5 access is denied" error on Windows 10.

```
import pyautogui

pyautogui.click() # click to put drawing program in focus
distance = 200
while distance > 20:
    print(distance, 0)
    pyautogui.dragRel(distance, 0, duration=0.1)  # move right
    distance = distance - 20
    print(0, distance)
    pyautogui.dragRel(0, distance, duration=0.1)  # move down
    print(-distance, 0)
    pyautogui.dragRel(-distance, 0, duration=0.1)  # move left
    distance = distance - 20
    print(0, -distance)
    pyautogui.dragRel(0, -distance, duration=0.1)  # move up

```

**Comment by asweigart** [comment]
This should be fixed in 0.9.34, but I'm going to keep this open until I can confirm on a Windows 10 box.

**Comment by asweigart** [comment]
Confirmed.
